iOS/Flows/Conversation/Cells/TimeMachineCollectionViewLayout.swift

```diff
@@ -126,7 +157,7 @@ class TimeMachineCollectionViewLayout: UICollectionViewLayout {
             let currentSection = indexPath.section
             let currentItemIndex = indexPath.item
 
-            var startZ: CGFloat = 0
+            var startZ: CGFloat = CGFloat(sortedItemsIndex) * self.itemHeight
```

Comment:
Fixes an issue where the second to last oldest message might look smaller than usual.